src/actions/transformCaseAction.ts

```diff
@@ -22,37 +22,37 @@ module Plywood {
 
     static fromJS(parameters: ActionJS): TransformCaseAction {
       var value = Action.jsToValue(parameters);
-      value.transformCaseType = parameters.transformCaseType;
+      value.transformType = parameters.transformType;
```

Comment:
the name was `transformType` before this PR, I just reverted it to the old ways (so there should be no net diff in `transformCaseAction` with this PR